test/endpoint_tests/test_graphql.py

```diff
@@ -0,0 +1,153 @@
+from boxwise_flask.db import db
+from boxwise_flask.models import Camps, Cms_Users
+
+
+def test_all_bases(client):
+    """example test for graphql endpoint"""
+
+    db.connect_db()
+    Camps.create(id=1, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=2, organisation_id=1, name="some text1", currencyname="hello")
```

Comment:
Can you use different names and currencies for the samples camps? Otherwise we would not notice if due to some bug only the first name would by returned.